docs/modules/ROOT/nav.adoc

```diff
@@ -52,7 +52,7 @@
 ** xref:scaling/binding.adoc[Binding]
 * xref:traits:traits.adoc[Traits]
 // Start of autogenerated code - DO NOT EDIT! (trait-nav)
-** xref:traits:3scale.adoc[3scale]
+** xref:traits:3scale.adoc[3Scale]
```

Comment:
It should be `3scale` (small 's'). FYI https://www.3scale.net/

Reply:
I changed it back, but that means there's a small bug in the generation script.

Reply:
If that was autogenerated, I'd suggest to keep it the way it was and report an issue to fix the autogeneration later

Reply:
Issue created #3972 .